### DISCUSSION TITLE
Purge CSS configured twice (in PostCSS + Tailwind config files)

### DISCUSSION THREAD

**Discussion by KostaVuckovic** [tailwindlabs/tailwindcss#2104] [Help] [answered]
Hello, I'm building Vue movie app with TailwindCSS and everything is fine when I open in localhost, but when I build for production with 'npm run build' and get that 'dist' folder and upload folder to netlify e.g. , style doesnt show up. I followed all steps in official site and still don't get it. Here is my public git repository:  https://github.com/KostaVuckovic/vue-movie-app

**Comment by estevanmaito** [comment, marked as answer]
You're purging two times and that's probably the cause.

You're configuring `purge` inside `postcss.config.js` https://github.com/KostaVuckovic/vue-movie-app/blob/master/postcss.config.js

And then in your `tailwind.config.js` https://github.com/KostaVuckovic/vue-movie-app/blob/master/tailwind.config.js

The first time you are purging the `.vue` files, while in the second you're purging (two times) only `.html`.

I didn't run your code, but if you stick with one config (chose PostCSS or Tailwind) and pass `purge` the right paths, it will probably fix the problem.

If you go for `tailwind.config.js`, `purge` would look like this:

```js
purge: [
    content: [
    './src/**/*.html',
    './src/**/*.vue',
    './src/**/*.js',
  ],
],
```

**Reply by KostaVuckovic** [reply]
Ahh yes, thank you so much :)
